### PR TITLE
Updated the toggle component to fix the accessibility issue for label…

### PR DESCRIPTION
**Pull request by maheshkanchetti**
… and checkbox

Closes IBM/carbon-components-angular#

{{short description}}

#### Changelog

**New**

* {{new thing}}

**Changed**

* {{change thing}}

**Removed**

* {{removed thing}}

### CODE REVIEW

File: src/toggle/toggle.component.ts [hunk 1]
```diff
@@ -109,6 +109,16 @@ export class Toggle extends Checkbox {
 		return this._offValues.value;
 	}
 
+	/**
+	 * Sets the aria labelledby on the `toggle`s label element.
+	 */
+	@Input() ariaLabelledby: string;
```
Comment (Donisius) [inline src/toggle/toggle.component.ts:115]: I believe the checkbox component has these inputs, does setting the inputs still work without having these?

File: src/ui-shell/header/header-action.component.ts [hunk 2]
```diff
@@ -16,7 +16,6 @@ import {
 			[ngClass]="{
 				'bx--header__action--active': active
 			}"
-			[attr.aria-label]="title"
```
Comment (Donisius) [inline src/ui-shell/header/header-action.component.ts:19]: Why does this need to be taken out?

File: src/ui-shell/sidenav/sidenav.component.ts [hunk 3]
```diff
@@ -17,7 +17,7 @@ import { NavigationItem } from "../header/header-navigation-items.interface";
 @Component({
 	selector: "ibm-sidenav",
 	template: `
-		<nav class="bx--side-nav__navigation" role="navigation">
+		<nav class="bx--side-nav__navigation" role="navigation" [attr.aria-label]="ariaLabel">>
```
Comment (Donisius) [inline src/ui-shell/sidenav/sidenav.component.ts:20]:
```suggestion
		<nav class="bx--side-nav__navigation" role="navigation" [attr.aria-label]="ariaLabel">
```

Comment (Donisius) [reply src/ui-shell/sidenav/sidenav.component.ts:20]: Also should probably be on multiple lines

File: src/tabs/tab-header-group.component.ts [hunk 4]
```diff
@@ -35,7 +35,7 @@ import { Subscription } from "rxjs";
 			<a
 				href="javascript:void(0)"
 				class="bx--tabs-trigger-text"
-				tabindex="-1">
+				tabindex="-1" *ngIf="(!getSelectedTab().headingIsTemplate && getSelectedTab().heading != '') || getSelectedTab().headingIsTemplate">
```
Comment (Donisius) [inline src/tabs/tab-header-group.component.ts:38]:
```suggestion
				tabindex="-1"
				*ngIf="(!getSelectedTab().headingIsTemplate && getSelectedTab().heading != '') || getSelectedTab().headingIsTemplate">
```

**Review by Donisius — CHANGES_REQUESTED** [review]
Thanks for taking this on! Just a few things

File: src/ui-shell/header/header-action.component.ts (hunk 2 above)
Comment (maheshkanchetti) [reply src/ui-shell/header/header-action.component.ts:19]: Header element with label as banner was reported as violation

File: src/ui-shell/sidenav/sidenav.component.ts (hunk 3 above)
Comment (maheshkanchetti) [reply src/ui-shell/sidenav/sidenav.component.ts:20]: yes, thanks

File: src/ui-shell/header/header-action.component.ts (hunk 2 above)
Comment (maheshkanchetti) [reply src/ui-shell/header/header-action.component.ts:19]: Facing the accessibility issue for that label 
Accessible name does not match or contain the visible label text

File: src/toggle/toggle.component.ts (hunk 1 above)
Comment (maheshkanchetti) [reply src/toggle/toggle.component.ts:115]: Thanks, we have overlooked it.
We are removing them from toggle as we have them in checkbox component

**Review by cal-smith — APPROVED** [review]
Thanks for taking this on 👍 everything seems good to me

File: src/ui-shell/header/header-action.component.ts (hunk 2 above)
Comment (Donisius) [reply src/ui-shell/header/header-action.component.ts:19]: Okay, that makes sense to me

**Pull request merged**